scripts/setup-obproxy.sh

```diff
@@ -225,6 +263,9 @@ spec:
               name: "sql"
             - containerPort: 2884
               name: "prometheus"
+          envFrom:
+            - configMapRef:
+                name: $CONFIG_MAP
```

Comment:
what if CONFIG_MAP is empty

Reply:
The CONFIG_MAP can not be empty. If `--cm` is given and is empty the script will throw an error. If `--cm` and `--env` are all given, it throws an error. If only `--env` is given, the script create a new config map.

Reply:
but what if none of --env or --cm is passed, I suspect the config map is only created when either of these parameters is passed